custom_components/econet300/binary_sensor.py

```diff
@@ -31,34 +33,69 @@ class EconetBinarySensorEntityDescription(BinarySensorEntityDescription):
         key="pumpCWUWorks",
         name="Water pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
         device_class=BinarySensorDeviceClass.RUNNING
     ),
     EconetBinarySensorEntityDescription(
         availability_key="pumpCirculation",
         key="pumpCirculationWorks",
         name="Circulation pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
         device_class=BinarySensorDeviceClass.RUNNING
     ),
     EconetBinarySensorEntityDescription(
         availability_key="pumpFireplace",
         key="pumpFireplaceWorks",
         name="Fireplace pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
         device_class=BinarySensorDeviceClass.RUNNING
     ),
     EconetBinarySensorEntityDescription(
         availability_key="pumpSolar",
         key="pumpSolarWorks",
         name="Solar pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
+        device_class=BinarySensorDeviceClass.RUNNING
+    ),
+    EconetBinarySensorEntityDescription(
+        availability_key="pumpCO",
```

Comment:
I don't know how to name it, It boiler pump
![image](https://github.com/pblxptr/ecoNET-300-Home-Assistant-Integration/assets/37038426/645a2b33-893a-42bf-890d-6380ad2b3a2d)